fe/fe-core/src/main/java/com/starrocks/qe/ExecuteScriptExecutor.java

```diff
@@ -33,6 +35,29 @@ public class ExecuteScriptExecutor {
     private static final Logger LOG = LogManager.getLogger(ExecuteScriptExecutor.class);
 
     public static void execute(ExecuteScriptStmt stmt, ConnectContext ctx) throws UserException {
+        if (stmt.isFrontendScript()) {
+            executeFrontendScript(stmt, ctx);
+        } else {
+            executeBackendScript(stmt, ctx);
+        }
+    }
+
+    private static void executeFrontendScript(ExecuteScriptStmt stmt, ConnectContext ctx) throws UserException {
```

Comment:
For security reason, we beteter add a static configuration to disable this feature when needed.

Reply:
added